### ISSUE TITLE
macOS M1 installation issues

### ISSUE THREAD

**Issue by jasoneveleth** [arximboldi/lager#181]
I cloned 8fd338d6d67dd2efc4f38c0634c2f849a7db8774. Tried to install, got:

CMake Error at cmake_install.cmake:41 (file):
  file INSTALL cannot find "/Users/jason/src/lager/resources/gui/gui.css": No
  such file or directory.

I fixed this by editing line 41 of cmake_install.cmake to be <path>/gui.scss. That fixed it. I think there is some configuration issue generating the css path name rather than scss.

I'm not sure how to fix it in the repo, but I thought I'd share.

**Comment by jasoneveleth** [comment]
Just to be clear, it works after that change.

**Comment by arximboldi** [comment]
No, sadly the `gui.css` file is required. However it should be generated from the `.scss` file properly. Maybe there are some missing targets in CMake to trigger that. In that folder there is a `make` file to do it.
